namnder/eventorganet/d-festeriet/body_sv.md

```diff
@@ -0,0 +1,15 @@
+# D-festeriet
+
+D-festeriet arbetar för att stärka gemenskapen som samtliga sektionsmedlemmar känner med varandra, samt andra studenter. Nämnden finns även som en hjälpresurs för att avlasta nämnder under gasqueverksamhet.
+
+Till nämndens uppgifter hör att - Anordna två skiftesgasquer per år, en efter Glögg-SM och en efter Val-SM. Syftet med skiftesgasquerna är att hedra sektionens avträdande och tillträdande funktionärer. - Anordna regelbundna teambuildings för sektionens funktionärer. - Eventuellt anordna andra sittningar och evenemang tillsammans med andra sektioner och studentkårer för att representera sektionen utåt.
```

Comment:
Same as for the English page, seems to be a leftover from the previous text in `Övriga funktionärer` though

```suggestion
Till nämndens uppgifter hör att:

- Anordna två skiftesgasquer per år, en efter Glögg-SM och en efter Val-SM. Syftet med skiftesgasquerna är att hedra sektionens avträdande och tillträdande funktionärer.

 - Anordna regelbundna teambuildings för sektionens funktionärer.

- Eventuellt anordna andra sittningar och evenemang tillsammans med andra sektioner och studentkårer för att representera sektionen utåt.
```

I would suggest adding the text about halvtidsfesten etc in the list as well